developer_manual/core/configfile.rst

```diff
@@ -117,17 +114,17 @@ ownCloud can be set to use a custom app directory in /config/config.php. Customi
 
 .. code-block:: php
 
-  'apps_paths' => [
-      0 => [
-          'path' => OC::$SERVERROOT.'/apps',
-          'url' => '/apps',
-          'writable' => true,
-      ],
-      1 =>[
-          'path' => OC::$SERVERROOT.'/apps-external',
-          'url' => '/apps-external',
-          'writable' => false,
-      ],
-  ],
-
-ownCloud will use the first app directory which it finds in the array with 'writable' set to true.
+    'apps_paths' => [
+        [
+            'path' => '/var/www/owncloud/apps',
```

Comment:
Are you sure this change and the one to L124 is a good idea? It then makes the installation location-specific. The previous version looks better.